app/controllers/concerns/authenticable_user.rb

```diff
@@ -8,8 +8,12 @@ def current_user
     User.find_by(id: payload["sub"])
   end
 
+  def authorization_header
+    Hash[*request.headers["Authorization"].to_s.split(" ")]
+  end
+
   def token
-    @token ||= request.headers["Authorization"].to_s.split(" ").last
+    @token ||= authorization_header["Bearer"]
```

Comment:
Would it be better or readable to use regexp matching?
```suggestion
    @token ||= request.headers["Authorization"].to_s.match(/Bearer (.*)/).to_a.last
```

Reply:
yes